packages/react-components/src/components/AppFrame/components/NavigationItem/NavigationItem.tsx

```diff
@@ -44,6 +44,7 @@ export const NavigationItem: React.FC<INavigationItemProps> = ({
   isActive,
   onClick,
   className,
+  isMobile,
```

Comment:
Do we need to add this prop? Can't we imply this from its wrapper `MobileNavigation`?

Reply:
Moved that flag to the `AppFrameContext`